help/en-US/Microsoft.PowerShell.SecretManagement.dll-Help.xml

```diff
@@ -947,7 +1068,18 @@ The secret value must be one of five supported types:</maml:para>
           </dev:type>
           <dev:defaultValue>None</dev:defaultValue>
         </command:parameter>
-        <command:parameter required="true" variableLength="true" globbing="false" pipelineInput="True (ByValue)" position="1" aliases="none">
+        <command:parameter required="false" variableLength="true" globbing="false" pipelineInput="False" position="named" aliases="none">
+          <maml:name>NoClobber</maml:name>
+          <maml:Description>
+            <maml:para>{{ Fill NoClobber Description }}</maml:para>
```

Comment:
I believe this is the PlatyPS template placeholder, may need to fill this description in.